app/views/candidates/home/index.html.erb

```diff
@@ -0,0 +1,79 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+
+    <h1 class="govuk-heading-xl">Request school experience placements</h1>
+
+    <p>
```

Comment:
In the design system docs, regular paragraphs are marked up with `<p class="govuk-body">`.

Additionally, I think this opening paragraph might benefit from being marked up as a lead paragraph with `<p class="govuk-body-l">`

Reply:
I've turned global styles on (I think Richard reviewed that PR - see `application.scss`)

This means all paragraphs get `govuk-body` unless overridden.

I've added the lead to the first paragraph though, good call on that